code.R

```diff
@@ -0,0 +1,69 @@
+#  load libraries; plyr must be loaded before tidyverse because plyr masks the count function from dplyr
+library(plyr)
+library(tidyverse)
+library(ggplot2)
+library(skimr)
+
+#  trying to use the Peng & Matsui EDA checklist, minus the steps that are not applicable
+
+## 1 formulate question ----
+## we are examining whether player skin color predicts the frequency of getting yellow carded
+
+## 2 load in data ----
+dataf <- read_csv(file.path('data', 'CrowdstormingDataJuly1st.csv'))
+
+## 4 look at top & bottom of data ----
+skim(dataf)
+head(dataf)
+
+#  omit missing data
+#  create average rating of player skin color
+dataf2 <- dataf %>% 
+  na.omit %>% 
+  mutate(rater_average = ((rater1 + rater2) / 2)) 
+
+skim(dataf2)
+count(dataf2, playerShort)
+#  1419 unique players
+count(dataf2, rater_average)
+#  very uneven distribution of skin colors in foosball (it's the devil!)
+
+#  creating a new df that combines rows based on player name 
+#  (so some players aren't overrepresented in the analysis based on how long they've been playing),
+#  that player's mean number of yellow cards per game, 
+#  and that player's rated skin color
+dataf3 <- ddply(dataf2, .(playerShort), summarise, mean_YC = mean(yellowCards))
```

Comment:
Reliability/ Maintainability
The code is condensed and is a little hard to follow. It would help to have some more explanation /documentation  for each line of code in this section. Would help if data frame names were more meaningful.